README.md

```diff
@@ -178,6 +178,6 @@ $client->setImpersonateUser(null);
 ### Thanks!
 
 * Thanks to [Thomas Spycher](https://github.com/tspycher/) for the 1st version of the class.
-* Thanks to [Thibault Duplessis aka. ornicar](https://github.com/ornicar) for the php-github-api library, great source of inspiration!
+* Thanks to [Thibault Duplessis aka. ornicar](https://github.com/ornicar) for the PHP-GitHub-API library, a great source of inspiration!
```

Comment:
:-1: 
The name of the library is "php-github-api" , see https://github.com/ornicar/php-github-api